### DISCUSSION TITLE
stealing mechanic via kubejs

### DISCUSSION THREAD

**Discussion by Quadruplay** [BeansGalaxy/Beans-Backpacks-2#206] [Feedback]
I want to make it so players can steal from other players while they are asleep. my plan is to add beans backpacks to the modpack and remove their recipes to have them act purely as proxies. then if a player falls asleep, I equip a backpack on them and shift their inventory to the backpack, letting other players steal from it. when they wake up everything from the backpack is shifted back to the inventory and the backpack is removed.

is this doable with this mod?

**Comment by BeansGalaxy** [comment]
I wouldn't think so. You could make it invisible by replacing the texture too. I'm not sure how easy it would be to move the items from their inventory to the backpack and keep those items you moved synced if someone does take something.
